capstone/feed-fetching/upload.sh

```diff
@@ -0,0 +1,7 @@
+scp -r ../feed-fetching wazuh-agent:~/cron/feed-fetching
+ssh wazuh-agent << EOF
+    cd ~/cron/feed-fetching
+    python3 -m virtualenv venv
+    source venv/bin/activate
```

Comment:
And what if the virtualenv is already created and we only want to update the script?

Reply:
Then it will leave existing virtualenv and install only new libraries from requirements